### PR TITLE
Picture factory

### PR DESCRIPTION
**Pull request by anxuae**

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -70,6 +70,9 @@ Default configuration
     # Crop each capture border in order to fit the paper size
     captures_cropping = False
 
+    # Thick (in pixels) between captures and picture borders/texts
+    margin_thick = 100
```
Comment (werdeil) [inline docs/config.rst:74]: I have an issue with this setting, if I write 1000 ou 2000, the pibooth crashes...

```
pibooth/pibooth/pictures/factory.py", line 451, in _image_resize_keep_ratio
    image = cv2.resize(cropped, (max_w, max_h), interpolation=inter)
cv2.error: OpenCV(4.1.1) /tmp/opencv-20190901-52549-14x6tkx/opencv-4.1.1/modules/imgproc/src/resize.cpp:3723: error: (-215:Assertion failed) inv_scale_x > 0 in function 'resize'
```

**Comment by anxuae** [comment]
* pibooth_qrcode is an error (oups)
* ok, I see, values are out of bound

**Comment by anxuae** [comment]
In fact, several values in the configuration are not checked (types and/or bound).
Maybe integrity checks should be done at the configuration level, not in each location where the option is accessed. What do you think?

File: docs/config.rst (hunk 1 above)
Comment (werdeil) [reply docs/config.rst:74]: Will be corrected by #135

**Comment by werdeil** [comment]
Merged as integrity will be done by #135

**Pull request merged**